src/client/components/LeaderboardList/AmountOfTime.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+export default function AmountOfTime({ timeSpent }) {
+  const hours = Math.floor(timeSpent / 3600);
+  const minutes = Math.floor((timeSpent % 3600) / 60);
+  const seconds = Math.floor((timeSpent % 3600) % 60);
+
+  if (!timeSpent) {
+    return (
+      <div className="timer-div">
+        <div>timer is requierd</div>
+      </div>
+    );
+  }
+
+  return (
+    <div className="timer-div">
+      <p className="timer">
+        {hours} hour {minutes} min {seconds} sec
```

Comment:
In lines 10:11 you used 2 consecutive divs, and here in line 17:18, you did a div > p tags. Wouldn't be better to choose one or the other? To be consistent.

Reply:
i put everything in a div so we have the possibility to style it better i guess but not sure